### ISSUE TITLE
Example in the metastore documentation doesn't work

### ISSUE THREAD

**Issue by sighingnow** [v6d-io/v6d#821]
Describe your problem
---------------------

Documentation link: https://v6d.io/notes/divein.html#vineyard-object

![image](https://user-images.githubusercontent.com/7144772/177088838-0d806fd8-03ef-453d-9fda-275cb86f308d.png)

Additional context
------------------

Thank Sitan Liu for raising the issue!

**Comment by liusitan** [comment]
OS: 
Macos Monterey 12.3
Python Dependency: 
running environment: absl-py==1.1.0
airflow-provider-vineyard==0.6.0
alabaster==0.7.12
alembic==1.8.0
anyio==3.6.1
apache-airflow==2.3.3rc1
apache-airflow-providers-ftp==3.0.0
apache-airflow-providers-http==3.0.0
apache-airflow-providers-imap==3.0.0
apache-airflow-providers-sqlite==3.0.0
apispec==3.3.2
argcomplete==2.0.0
astunparse==1.6.3
attrs==20.3.0
Babel==2.10.3
black==22.6.0
blinker==1.4
bokeh==3.0.0.dev9
breathe==4.34.0
cachelib==0.9.0
cachetools==5.2.0
cattrs==1.10.0
certifi==2022.6.15
cffi==1.15.1
charset-normalizer==2.1.0
click==8.1.3
clickclick==20.10.2
cloudpickle==2.1.0
colorama==0.4.5
colorlog==4.8.0
commonmark==0.9.1
connexion==2.14.0
cron-descriptor==1.2.30
croniter==1.3.5
cryptography==37.0.3
dask==2022.6.1
Deprecated==1.2.13
dill==0.3.5.1
distributed==2022.6.1
dnspython==2.2.1
docutils==0.17.1
email-validator==1.2.1
etcd-distro==3.5.1
flake8==4.0.1
Flask==2.1.2
Flask-AppBuilder==4.1.2
Flask-Babel==2.0.0
Flask-Caching==2.0.0
Flask-JWT-Extended==4.4.2
Flask-Login==0.6.1
Flask-Session==0.4.0
Flask-SQLAlchemy==2.5.1
Flask-WTF==0.15.1
flatbuffers==1.12
fsspec==2022.5.0
future==0.18.2
gast==0.4.0
google-auth==2.9.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
graphviz==0.8.4
greenlet==2.0.0a2
grpcio==1.47.0
gunicorn==20.1.0
h11==0.12.0
h5py==3.7.0
HeapDict==1.0.1
httpcore==0.13.7
httpx==1.0.0b0
idna==3.3
imagesize==1.4.1
importlib-metadata==4.12.0
inflection==0.5.1
isort==5.10.1
itsdangerous==2.1.2
Jinja2==3.1.2
jsonschema==4.6.1
keras==2.9.0
Keras-Preprocessing==1.1.2
lazy-object-proxy==1.7.1
libclang==14.0.1
linkify-it-py==2.0.0
locket==1.0.0
lockfile==0.12.2
Mako==1.2.1
Markdown==3.3.7
markdown-it-py==2.1.0
MarkupSafe==2.1.1
marshmallow==3.17.0
marshmallow-enum==1.5.1
marshmallow-oneofschema==3.0.1
marshmallow-sqlalchemy==0.26.1
mccabe==0.6.1
mdit-py-plugins==0.3.0
mdurl==0.1.1
msgpack==1.0.4
mxnet==2.0.0b1
mypy-extensions==0.4.3
numpy==1.23.0
oauthlib==3.2.0
opt-einsum==3.3.0
packaging==21.3
pandas==1.4.3
parsec==3.13
partd==1.2.0
pathspec==0.9.0
pendulum==2.1.2
Pillow==9.2.0
platformdirs==2.5.2
pluggy==1.0.0
prison==0.2.1
protobuf==3.19.4
psutil==5.9.1
pyarrow==8.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycodestyle==2.8.0
pycparser==2.21
pyflakes==2.4.0
Pygments==2.12.0
PyJWT==2.4.0
pyparsing==3.0.9
pyrsistent==0.18.1
python-daemon==2.3.0
python-dateutil==2.8.2
python-nvd3==0.15.0
python-slugify==6.1.2
pytz==2022.1
pytzdata==2020.1
PyYAML==6.0
requests==2.28.1
requests-oauthlib==1.3.1
rfc3986==1.5.0
rich==12.4.4
rsa==4.8
scipy==1.9.0rc1
setproctitle==1.2.3
six==1.16.0
sniffio==1.2.0
snowballstemmer==2.2.0
sortedcontainers==2.4.0
Sphinx==5.0.2
sphinx-rtd-theme==1.0.0
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
SQLAlchemy==1.4.9
SQLAlchemy-JSONField==1.0.0
SQLAlchemy-Utils==0.38.2
swagger-ui-bundle==0.0.9
tabulate==0.8.10
tblib==1.7.0
tenacity==8.0.1
tensorboard==2.9.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.9.1
tensorflow-estimator==2.9.0
tensorflow-io-gcs-filesystem==0.26.0
termcolor==1.1.0
text-unidecode==1.3
tomli==2.0.1
toolz==0.11.2
torch==1.12.0
tornado==6.2b2
treelib==1.6.1
typing_extensions==4.3.0
uc-micro-py==1.0.1
unicodecsv==0.14.1
urllib3==1.26.9
vineyard==0.6.0
vineyard-dask==0.6.0
vineyard-ml==0.6.0
Werkzeug==2.1.2
wrapt==1.14.1
WTForms==2.3.3
xgboost==1.6.1
xyzservices==2022.6.0
zict==2.2.0
zipp==3.8.0
M

**Comment by sighingnow** [comment]
Thanks @liusitan for the information.

**Comment by sighingnow** [comment]
Addressed by #827
